use-timescale/metrics-logging/metrics-to-prometheus.md

```diff
@@ -10,65 +10,102 @@ cloud_ui:
-    -e DATA_SOURCE_NAME="postgres://monitoring:<PASSWORD>@<TIMESCALE-HOST:PORT>/tsdb?sslmode=require" \
-    quay.io/prometheuscommunity/postgres-exporter
-    ```
-    To reduce latency and potential data transfer costs,  best practice is to run `postgresql_exporter` in the same AWS region as your Timescale Cloud service.
+1. **Create a user to scrape the metrics**
+
+   1. Create a user named `monitoring` with a password:
+
+      ```sql
+      CREATE USER monitoring WITH PASSWORD '<password>';
+      ```
+
+   1. Grant the `pg_read_all_stats` permission to the `monitoring` user:
+
+      ```sql
+      GRANT pg_read_all_stats to monitoring;
+      ```
+
+1. **Install PostgreSQL Exporter**
 
-1. Once the postgresql_exporter is up and running, and successfully connected to the Timescale service, you can configure your Prometheus server to scrape the postgresql_exporter metrics endpoint. This endpoint exposes all the metrics provided by the exporter. 
+   PostgreSQL Exporter collects PostgreSQL performance metrics and exposes them in a Prometheus-compatible format. [Install PostgreSQL Exporter][install-exporter] on the host that you use to connect to your $SERVICE_SHORT and collect telemetry. To reduce latency and potential data transfer costs, run PostgreSQL Exporter in the same AWS region as your $SERVICE_LONG.
+
+   For example, install using Docker:
+
+   1. Pull the Docker image:
+   
+      ```bash
+      docker pull quay.io/prometheuscommunity/postgres-exporter
+      ````
+
+   1. Run the container:
 
-    To view all the exposed metrics, execute curl on the same host:
     ```bash
-    curl localhost:9187/metrics
+    docker run -d --name=postgresql_exporter \
+    -e DATA_SOURCE_NAME="postgresql://<username>:<password>@<host>:<port>/<database>?sslmode=require" \
+    -p 9187:9187 quay.io/prometheuscommunity/postgres-exporter
     ```
 
-## Grafana dashboard for PostgreSQL metrics
+   - `<username>`: `monitoring`
+   - `<password>`: the `monitoring` user password
+   - `<host>`, `<port>`, and `<database>`: configure using your [connection details][connection-info].
+
+   If not using Docker, download the binary and configure the `DATA_SOURCE_NAME` environment variable similarly. To check the installation, navigate to `http://<exporter-host>:9187/metrics`. You should see PostgreSQL metrics in the Prometheus format.
+
+1. **Configure Prometheus to scrape metrics**
+
+   1. Update the `prometheus.yml` file to include PostgreSQL Exporter as a scrape target. If `prometheus.yml` has not been created during installation, create it manually:
+
+      ```yaml
+      global:
+        scrape_interval: 15s
+      
+      scrape_configs:
+      - job_name: 'postgresql'
+        static_configs:
+         - targets: ['<exporter-host>:9187'] 
+      ```
+
+      Replace `<exporter-host>` with the hostname or IP address of the PostgreSQL Exporter. 
+
+   1. Restart Prometheus.
+
+   1. Check the Prometheus UI at `http://<prometheus-host>:9090`.
```

Comment:
```suggestion
   1. Connect to the Prometheus UI at `http://<prometheus-host>:9090`.
```

Reply:
This is in the sense of "open a browser window and look at the UI". Why would we say "connect" here? Sounds like there are extra steps involved.